modules/Quiz/resources/quiz.js

```diff
@@ -856,16 +871,24 @@
             var _this = this;
             $.each(cPo.answeres, function (key, value) {
                 if (key == $.cpObject.cpArray[questionNr].selectedAnswer) {
-                    $('#' + key).parent().addClass("wide single-answer-box-bk-apply disable");
-                    $('#' + key).removeClass('single-answer-box-txt')
-                        .addClass(function(){
-                            // transform answer element so it appears as an already-answered on with "applied" element
-                            $(this).addClass('single-answer-box-txt-wide ')
-                                .after($('<div></div>') // adding continue/applied div as button
-                                    .addClass("single-answer-box-apply qApplied disable").attr('aria-disabled', true).attr('role', 'button')
-                                    .text(gM('mwe-quiz-selected'))
-                            );
+                    if (cPo.submitRequested) {
```

Comment:
Do we set submitRequested to false at any point (other than initial) ? 
Do we need to?

Reply:
I think we shouldn't care about it, because "submitRequested" flag creates and works only for the current quiz session. In case if API respond with delay - question get "isAnswerd" flag then "submitRequested" flag become unnecessary.